### PR TITLE
MAT-7501 support QICore v6.0.0

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-7501](https://jira.cms.gov/browse/MAT-7501)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: .github/workflows/unit_test_coverage.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
         run: npm ci
 
       - name: Audit dependencies for security vulnerabilities
-        run: npm audit --production
+        run: npm audit --omit=dev --audit-level=${{ vars.NPM_AUDIT_LEVEL }}
```
Comment (adongare) [inline .github/workflows/unit_test_coverage.yml]: this can be reverted now if this is related to axios. axios version with the fix is available now

Comment (sb-cecilialiu) [reply .github/workflows/unit_test_coverage.yml]: @adongare Thanks Ashok! Reverted.

Comment (adongare) [reply .github/workflows/unit_test_coverage.yml]: thanks @sb-cecilialiu. Need to update axios dependency as well `"axios": "^1.7.4",` to use vulnerability free version

Comment (sb-cecilialiu) [reply .github/workflows/unit_test_coverage.yml]: @adongare yeah sure, just updated it.

**Review by adongare — APPROVED** [review]

**Pull request merged**